mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -537,11 +701,17 @@
         <GlossaryCategory value=""/>
         <GlossaryWeakArray index="0" value="Marauder"/>
         <GlossaryWeakArray index="1" value="Zergling"/>
+        <CardLayouts index="0">
+            <LayoutButtons index="12" Face="Cancel" AbilCmd="BypassArmorCancel,255" Row="2" Column="4"/>
```

Comment:
Steady Targeting (Snipe) can now be manually canceled

This adds a cancel button and shortcut to the command card